pkg/frontend/src/App.tsx

```diff
@@ -83,31 +88,31 @@ export default function App() {
                                 <Divider/>
                                 <List>
                                     <ListItem key="Claims" disablePadding>
-                                        <ListItemButton component={RouterLink} to="/claims">
+                                        <ListItemButton component={RouterLink} to="/claims" sx={{ backgroundColor: isLinkActive('/claims') ? '#312e67' : 'transparent' }}>
```

Comment:
The duplication does not look good. 
Can you extract function that would return full sx style? Because I'd like to get the style similar to Komodor app.

Reply:
Implemented a function styleHighLight(path: string) and passing this function in sx style in ListItemButton. 
eg: <ListItemButton component={RouterLink} to="/claims" sx={styleHighLight("/claims")}>

I think this approach has  minimized code redundancy.

![Screenshot 2023-08-24 142027](https://github.com/komodorio/komoplane/assets/82561880/2e81dec6-47a9-4397-9516-f7515309e93c)